### PR TITLE
another doc creation - nickname docs

### PR DESCRIPTION
**Pull request by zAG6z**
again as always change if u need to

### CODE REVIEW

File: docs/commands/clear.md [hunk 1]
```diff
@@ -12,12 +12,20 @@ The minimum amount of messages you can delete at once is 2. The maximum you can
...
+>>>>>>> upstream/gh-pages
 * NOTE: Required Parameters = <> and Optional Parameters = []
 
 * Usage: `{prefix}clear <messagesToDelete>`
 
+<<<<<<< HEAD
+##Aliases
+=======
 ## Aliases
+>>>>>>> upstream/gh-pages
```
Comment (xf8b) [inline docs/commands/clear.md:28]: you can remove the section that has
```
<<<<< HEAD
<stuff>
=========
<other stuff>
>>>>>>> upstream
```
and keep the <other stuff>

**Review by xf8b — CHANGES_REQUESTED** [review]

**Comment by zAG6z** [comment]
all gone

File: docs/commands/clear.md [hunk 2]
```diff
@@ -18,6 +18,5 @@ The minimum amount of messages you can delete at once is 2. The maximum you can
 * Usage: `{prefix}clear <messagesToDelete>`
 
 ## Aliases
-* `purge`
```
Comment (xf8b) [inline docs/commands/clear.md:21]: why is this crob

File: docs/commands/nickname.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+## Usage
+ * NOTE: Required Parameters = <> and Optional Parameters = []
+ 
+Usage: `{prefix}nickname -m <@member/memberId/memberName> -n [newNick]`
+* NOTE: Leaving the `-n` flag empty removes the current nick.
+* Flags: member - `-m` or `--member`, nickname - `-n` or `--nickname`
+
+---
+## Aliases
+`nick`, `nickname`
```
Comment (xf8b) [inline docs/commands/nickname.md:23]: you can remove the `nickname` part as its not a alias but the actual command name
```suggestion
`nick`
```

**Review by xf8b — CHANGES_REQUESTED** [review]

File: docs/commands/clear.md (hunk 2 above)
Comment (zAG6z) [reply docs/commands/clear.md:21]: what

Comment (xf8b) [reply docs/commands/clear.md:21]: why you delete it

Comment (zAG6z) [reply docs/commands/clear.md:21]: i shouldnt have huh i dont remeber doing that

**Review by xf8b — APPROVED** [review]
yes

**Pull request merged**